tests_e2e/orchestrator/docker/Dockerfile

```diff
@@ -73,5 +73,6 @@ RUN \
     #                                                                                                            \
     echo 'export PYTHONPATH="$HOME/WALinuxAgent"' >> $HOME/.bash_profile                                      && \
     echo 'export PATH="$HOME/.local/bin:$PATH"' >> $HOME/.bash_profile                                        && \
+    echo 'cd $HOME' >> $HOME/.bash_profile                                                                    && \
```

Comment:
Every time I start an interactive container, I need to cd to home. Adding it here for convenience.

Reply:
(container image in the repo is updated)